.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
+node_js: 12
+install:
+  - npm install
+script:
+  - npm run lint
+  - npm run i18n_extract
+  - npm run test
+  - npm run build
+after_success:
+  - npx semantic-release
+  - npm run coveralls
```

Comment:
Just confirmed with @estute that we're trying to move to codecov instead of coveralls.